src/server/db/QueryModels/AttendanceModel.js

```diff
@@ -74,6 +74,14 @@ export default class AttendanceModel extends AttendanceQueries {
 
     await sendWarningEmails(arrayOfUser);
   }
+  async updateAttendanceStatus(data) {
```

Comment:
// try to destructure when possible to avoid giant declaration blocks
updateAttendanceStatus({ selectedStudent, selectedStatus, selectedDate })
const selectedName = selectedStudent.value
const timeString = selectedDate.slice(0, 10)